packages/arb-tx-aggregator/arbosmachine/arbosmachine.go

```diff
@@ -252,7 +253,7 @@ func handleDebugPrints(debugPrints []value.Value) {
 
 func (m *Machine) ExecuteAssertion(
 	maxSteps uint64,
-	messages []inbox.InboxMessage,
+	messages C.ByteSliceArray,
```

Comment:
We definitely don't want to expose Cgo implementation details to this higher level code